app/src/pages/inside/projectSettingsPageContainer/content/notifications/ruleGroup/ruleGroup.jsx

```diff
@@ -0,0 +1,330 @@
+
+  const handleRuleItemClick = (isShown) => {
+    if (isShown) {
+      trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.CLICK_TO_EXPAND_NOTIFICATIONS_DETAILS);
+    }
+  };
+
+  const confirmAdd = (newNotification) => {
+    const notification = convertNotificationCaseForSubmission(newNotification);
+    dispatch(addProjectNotificationAction(notification));
+  };
+
+  const confirmDelete = (id) => {
+    dispatch(deleteProjectNotificationAction(id));
+  };
+
+  const confirmEdit = (notification) => {
+    dispatch(
+      updateProjectNotificationAction(
+        convertNotificationCaseForSubmission({
+          ...notification,
+          name: notification.ruleName,
+        }),
+      ),
+    );
+  };
+
+  const onAdd = () => {
+    trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.CLICK_CREATE_RULE_BUTTON);
+
+    dispatch(
+      showModalAction({
+        id: 'addEditNotificationModal',
+        data: {
+          type: pluginName,
+          actionType: MODAL_ACTION_TYPE_ADD,
+          onSave: confirmAdd,
+          notification: DEFAULT_CASE_CONFIG,
+          notifications,
+        },
+      }),
+    );
+  };
+
+  const onEdit = (notification) => {
+    trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.CLICK_ICON_EDIT_NOTIFICATIONS);
+
+    dispatch(
+      showModalAction({
+        id: 'addEditNotificationModal',
+        data: {
+          actionType: MODAL_ACTION_TYPE_EDIT,
+          onSave: confirmEdit,
+          notification,
+          notifications,
+        },
+      }),
+    );
+  };
+
+  const onDelete = (notification) => {
+    trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.CLICK_ICON_DELETE_NOTIFICATIONS);
+
+    dispatch(
+      showModalAction({
+        id: 'deleteNotificationModal',
+        data: {
+          onSave: () => confirmDelete(notification.id),
+        },
+      }),
+    );
+  };
+  const onCopy = (notification) => {
+    trackEvent(PROJECT_SETTINGS_NOTIFICATIONS_EVENTS.CLICK_ICON_DUPLICATE_NOTIFICATIONS);
+
+    const { id, ...newNotification } = notification;
+    dispatch(
+      showModalAction({
+        id: 'addEditNotificationModal',
+        data: {
+          actionType: MODAL_ACTION_TYPE_COPY,
+          onSave: (withoutAttributes) => confirmAdd(withoutAttributes),
+          notification: {
+            ...newNotification,
+            ruleName: notification.ruleName + COPY_POSTFIX,
+          },
+          notifications,
+        },
+      }),
+    );
+  };
+
+  const actions = [
+    {
+      icon: CopyIcon,
+      handler: onCopy,
+      dataAutomationId: 'duplicateNotificationRuleIcon',
+    },
+    {
+      icon: PencilIcon,
+      handler: onEdit,
+      dataAutomationId: 'editNotificationRuleIcon',
+    },
+    {
+      icon: BinIcon,
+      handler: onDelete,
+      dataAutomationId: 'deleteNotificationRuleIcon',
+    },
+  ];
+
+  return (
+    <div className={cx('rule-section')}>
+      <Layout description={''} className={cx('rule-section-layout')}>
```

Comment:
Should description be empty?

Reply:
Yes, seems there is no need to add description, cause I've added separate section for heading